defaults/krill.conf

```diff
@@ -106,6 +106,102 @@
 #  <service_uri>...                       (various UI resources)
 ### service_uri = "https://localhost:3000/"
 
+
+######################################################################################
+#                                                                                    #
+#                                     METRICS                                        #
+#                                                                                    #
+######################################################################################
+
+# Krill exposes metrics for use with prometheus under: <service_uri>/metrics
+#
+# By default a lot of details are included in these metrics. Generating these metrics
+# is not particularly hard on Krill, but in case you have many CAs, children or publishers
+# under your Krill instance you may still want to disable certain metrics to avoid
```

Comment:
As we don't know that it can overwhelm Prometheus maybe instead of "to avoid overwhelming your prometheus instance with too much data" say "to reduce the amount of data fetched and stored by Prometheus" ?

Reply:
Should we recommend a fetch frequency? The Prometheus default is once a minute according to https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config. Does Krill cache the metric data and thus fetching more often than the cache period wouldn't make sense?

Reply:
sure wfm

Reply:
regarding fetch frequency: krill caches status for reporting so doesn't mind. You don't know when something might change.. as soon as something does the cache is updated. So, local policy.. no recommendation I think.